core/src/main/java/com/klaviyo/coresdk/networking/requests/KlaviyoRequest.kt

```diff
@@ -1,6 +1,8 @@
-package com.klaviyo.coresdk.networking
+package com.klaviyo.coresdk.networking.requests
 
-abstract class KlaviyoRequest: NetworkRequest() {
+import com.klaviyo.coresdk.networking.NetworkBatcher
+
+internal abstract class KlaviyoRequest: NetworkRequest() {
     companion object {
         internal const val BASE_URL = "https://a.klaviyo.com"
```

Comment:
Not sure if it matters, but is this something we could make configurable? I could see that being very useful for testing, like if we wanted to stick an ngrok url in here.

Reply:
Absolutely yes. I  was planning specify local URLs in the build variables and then inject those into the manifest file originally, but I think there were issues with doing this sort of manifest work with a lib.
There are multiple ways of achieving this though, we could also utilize BuildConfig, or write to a properties file on the system. It's definitely planned anyway